File: PRACTICA DE CAMPO 02-S04/PYTHON/s04_pc02_menu_interactivo.py
while True:
    opcion_menu = int(input("Indique opción: 1-Suma, 2-Resta, 3-Multiplicación, 4-División, 5-Salir :"))
    if opcion_menu == 1:
        a=int(input("Primer Número: "))
        b=int(input("Segundo Número: "))
        resultado=a+b
    elif opcion_menu == 2:
        a=int(input("Primer Número: "))
        b=int(input("Segundo Número: "))
        resultado=a-b
    elif opcion_menu == 3:
        a=int(input("Primer Número: "))
        b=int(input("Segundo Número: "))
        resultado=a*b
    elif opcion_menu == 4:
        a=int(input("Primer Número: "))
        b=int(input("Segundo Número: "))
        if b != 0:
            resultado=a/b
        else:
            print("No se puede dividir por cero.")
            continue
    elif opcion_menu == 5:
        print("Hasta la próxima.")
        break
    else:
        print("Opción invalida, vuelva a intentarlo.")
        continue
    print("Resultado: ", resultado)

File: PRACTICA DE CAMPO 02-S04/C#/PracticaCampo_02_S04/s04_pc02_menu_interactivo.cs
int opcion_menu = 0;
do{   
    Console.WriteLine("Indique opción: 1-Suma, 2-Resta, 3-Multiplicación, 4-División, 5-Salir");
    opcion_menu=int.Parse(Console.ReadLine());
    if(opcion_menu>=1 && opcion_menu<= 4){
        double resultado=0;
        Console.WriteLine("Primer número:");
        double primer_numero=double.Parse(Console.ReadLine());
        Console.WriteLine("Segundo número:");
        double segundo_numero=double.Parse(Console.ReadLine());
        switch (opcion_menu){
            case 1:
                resultado = primer_numero + segundo_numero;
                break;
            case 2:
                resultado = primer_numero - segundo_numero;
                break;
            case 3:
                resultado = primer_numero * segundo_numero;
                break;
            case 4:
                if (segundo_numero != 0){
                    resultado = primer_numero / segundo_numero; 
                }else{
                    Console.WriteLine("No se puede dividir con cero.");
                }
                break;
            //default:
                //Console.WriteLine("Opción no válida. Por favor, seleccione una opción entre 1 y 5.");
        }
        if(opcion_menu==4 && segundo_numero > 0){
            Console.WriteLine("Resultado: " + resultado);
        }else{
            Console.WriteLine("Resultado: " + resultado);
        }
        
    }else if (opcion_menu != 5){
        Console.WriteLine("Opción invalida, vuelva a intentarlo.");
    }
}while(opcion_menu != 5);

Console.WriteLine("Hasta la próxima.");